### ISSUE TITLE
Problem with feeds that contain "&". Results in "&amp"

### ISSUE THREAD

**Issue by EternalBlack** [verbb/feed-me#492]
### Description

Feeds that contain a `&` inside the XML result in `&amp;`. I.e. having a `&` inside the XML title of a feed and using that for the craft title results in (quite ugly) `&amp;` inside the title. This probably also is the reason for duplicate entries if the `title` is the unique identifier as `&` != `&amp;`.

### Steps to reproduce

1. Get a feed with a `&` inside the XML. Use the tag containing the `&` for a field of your choice.
2. Run the feed and and watch the outcome resulting in `&amp;` instead of just `&`.
3. If chose the field as unique identifier for re-runs of the feed it will result in duplicate entries as  `&` != `&amp;`.

### Additional info

- Plugin version: 3.1.15 Pro (just upgraded to pro)
- Craft version: 3.1.18 Pro

**Comment by engram-design** [comment]
I believe this is an issue with the XML itself. Do you have any control over the feed?

According to [this article](https://www.ibm.com/support/knowledgecenter/en/SSS28S_8.1.0/XFDL/i_xfdl_g_computes_illegal_characters_in_xml_attributes.html):
> "Note that the ampersand (&) and less-than (<) characters are not permitted in XML attribute values."

This also goes for the content between nodes as well.

If you have control over the feed's contents, it'd be best to wrap it in `CDATA` tags to escape the characters, like you would XML.

```
<![CDATA[Testing this & that]]>
```

You can also use `&amp;` as the value in your XML, which will be converted to a `&` through my tests.

```
<title>Testing this &amp; that</title>
```

The other option is to use JSON, but that's not always a viable option depending on where you get your feed from.

**Comment by EternalBlack** [comment]
Thank you for clarifying! We actually produced the this error ourself... So sorry for the accusation... We Had a `IncognitoField` which was disabled an gets fed the same information as the title filed. As the entry's title we used `{{ icognitoField }}`... This resulted in a HTML parsed result... producing the `&amp;` where after that feed-me (rightfully) though there was a difference in the title (as we used the title as unique identifier).
We "fixed" it by using `{{ icognitoField|raw }}` instead. Works fine now. Thank you for your time and sorry again.
